xapian-applications/omega/worker.h

```diff
@@ -83,8 +83,10 @@ class Worker {
 		 std::string& keywords,
 		 std::string& author,
 		 int& pages);
-    /** Returns and error message if the extraction fails, or and empty string
+    /** Returns an error message if the extraction fails, or an empty string
      *  if everything is okay.
      */
-    std::string get_error();
+    inline std::string get_error() const{
+	return error;
+    }
```

Comment:
You don't need to say `inline` here - a method defined within a class like this is implicitly `inline`.

Also, there's a missing space between `const` and `{`.

Reply:
I will remove that.